lifecycle_test.go

```diff
@@ -118,6 +127,18 @@ func createRb(t *testing.T, specName, sourceOption, rbName, rbVersion string, sy
 	assert.NoError(t, lcCli.Exec(argsAndOptions...))
 }
 
+func exportRb(t *testing.T, rbName, rbVersion string) {
+	output := lcCli.RunCliCmdWithOutput(t, "rbe", rbName, rbVersion)
```

Comment:
I suggest downloading to a specific location in order to also test the target directory option.
Then it might be better to verify the downloaded instead of relaying on the output of the command that might change (if I'm not mistaken, the command currently doesn't even have an output)